products/sle/main.pm

```diff
@@ -288,6 +290,10 @@ if (sle_version_at_least('15') && !check_var('SCC_REGISTER', 'installation')) {
                 $prefix .= "-Staging:" . get_var("STAGING");
             }
             my $module_repo_name = get_var("REPO_SLE${version}_MODULE_${repo_name}", "$prefix-Module-$full_name-POOL-$arch-Build$build-Media1");
+            # REPO_SLE* settings and repo names are different for products and modules
+            # Assign the proper repo name if current $short_name is not a module.
+            $module_repo_name = get_var("REPO_SLE_${repo_name}${version}_POOL", "$prefix-Product-$full_name-POOL-$arch-Build$build-Media1")
```

Comment:
hard to follow your logic in such form . suggesting the following 

```
my $repo_variable_name= is_module($short_name) ? "REPO_SLE_${repo_name}${version}_POOL" : "REPO_SLE${version}_MODULE_${repo_name}";
my $module_repo_name = get_var($repo_variable_name, "$prefix-Module-$full_name-POOL-$arch-Build$build-Media1") ;
```

Reply:
just a side note : we really overcomplicate things here . we have a variable which should define a repo name and we deciding about name of variable which should decide about name of a repo :\ . 
Not directly related to PR just thinking loud

Reply:
That's why we have this is_module method now, which makes it readable. I would rewrite these lines not to define variable twice and not with one liners as string is too long. But other than that looks good. 
As a side note, this thing was not supposed to stay too long, that's why it has WORKAROUND in the name of variable. So we may refactor it next time we introduce some changes.

Reply:
REPO_SLE${version}_MODULE_${repo_name} and REPO_SLE_${repo_name}${version}_POOL  
 doesn't have WORKAROUND in their names , so will logic selecting variable which select something will remain after you will remove workaround ? I have impression that yes . My point is that instead selecting which variable will select something we can just simply select something :D

Reply:
Line #272. Part which you are referring to is parsing that variable and uses repo variables not to have hardcoded paths. You are talking about long term plan. You can create a ticket for refactoring. I don't think it's needed for adding product via ftp.

Reply:
@asmorodskyi working on suggestion. will push after test runs complete.